app/src/main/java/edu/drexel/lapcounter/lapcounter/backend/Database/Workouts.java

```diff
@@ -0,0 +1,111 @@
+package edu.drexel.lapcounter.lapcounter.backend.Database;

```

Comment:
Since my code got pulled into master first, the Workouts Database code will conflict with mine as I had made some changes.  If you get yours to a state where the diff is just additions rather the conflict, it would make the merging much easier.

Also, i'd keep Workouts as workouts right now, we can update to Workout at a later date when the codebase is more stable